### ISSUE TITLE
A Video From An Assignment That Gives Wrong Information

### ISSUE THREAD

**Issue by MichaelYochpaz** [TheOdinProject/curriculum#13687]
Hey, just started learning the TOP course (which is great btw!) and I noticed one of the linked videos (which is also an assignment to watch) has some wrong / confusing information and it should be removed in my opinion (the [MDN article](https://developer.mozilla.org/en-US/Learn/Common_questions/How_does_the_Internet_work) which is also linked in the assignments is great and is more than enough).

[Video Link](https://www.youtube.com/watch?v=7_LPdttKXPc)
[TOP Assignment Link](https://www.theodinproject.com/courses/web-development-101/lessons/how-does-the-web-work?ref=lnav#assignment)

A. It says on [1:06](https://youtu.be/7_LPdttKXPc?t=66) & [1:36](https://youtu.be/7_LPdttKXPc?t=96) that the difference between a client and a server is that servers connect directly to the internet while clients connect through ISPs.
While some server providers may also be an ISP that's not always the case, and this is not the right way (at least in my opinion) to differentiate a server from a client, and "Your computer at home is not a server because it's not connected directly to the internet" is a bad way to differentiate a client from a server.

B. On [3:23](https://www.youtube.com/watch?v=7_LPdttKXPc&feature=youtu.be&t=203) it says & shows that every computer has a unique IP address and later shows that both "You" and "Boss" computers have different IP addresses even though they're connected to the same router.
While it's correct for local IP address, it seems like they refer to public IP address since they don't address the differences.
So while it's not completely wrong, if you'll pause & look at the diagram on [3:50](https://www.youtube.com/watch?v=7_LPdttKXPc&feature=youtu.be&t=230) it seems as if your computer / your boss's computer have the same IP address type as facebook.com or marketresearch.com and someone without knowledge might think that each computer has a unique public IP address even though all devices on the local network get the same public IP address as their router.

**Comment by mindovermiles262** [comment]
Hey @MichaelYochpaz 

You're completely correct - I wouldn't define a "client" as a computer not directly connected to the internet.

However, this lesson is designed to ELI5. It's just for understanding that the internet isn't "magic" and I feel that this resource is valid.  I don't think we want to dig into RFC1918 ajnd NAT Traversal this early in this lesson (or even in TOP).

If you have a different video that you think explains these concepts better I would happily swap them out, but I think this video is important and would like to keep it in the curriculum. It's a nice high-level understanding of how the internet works and has good visual representations.
